.tekton/tasks/update-repository.yaml

```diff
@@ -116,77 +118,22 @@ spec:
         git config --global safe.directory "${PWD}"
         git config --local user.email "$GIT_EMAIL"
         git config --local user.name "$GIT_USER"
+        git remote set-url origin "$(cat "$WORKSPACE_BASIC_AUTH_DIRECTORY_PATH/.git-credentials")"
 
         # Create branch
         git branch --copy --force "$COMMIT_BRANCH"
         git checkout "$COMMIT_BRANCH"
 
         # Run script
-        UPSTREAM_COMMIT=$(git rev-parse HEAD)
         "${SCRIPT_PATH}" "${SCRIPT_ARGS[@]}"
 
-        # Log changes
```

Comment:
If I read this correctly @Roming22 we were not even pushing at all before but building a list of the changes.

Is that correct?  

I think adding some additional details around this in the PR description about this, and *why* we want to move from building the log to a push, would be helpful.

Thanks.

Reply:
That's correct. I had issues pushing to the repo I thought that I would need to send the changes through the GitHub API, for which I needed the detailed information I was logging in `.commits.json`.

Since I've since then managed to perform a `git push` to the repository, the task can be greatly simplified.